src/e2e/java/uk/gov/hmcts/reform/sscs/functional/EvidenceShareFunctionalTestThree.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.sscs.functional;
```

Comment:
What is the reason we have file name EvidenceShareFunctionalTestThree and ..Two? can we please combine into one file if possible